tests/system/runtimes/test_databricks.py

```diff
@@ -161,14 +166,37 @@ def test_kwargs_from_file(self):
             handler="func",
             auto_build=True,
             project="databricks-proj",
-            params={
-                "task_parameters": {"timeout_minutes": 15},
-                "param1": "value1",
-                "param2": "value2",
-            },
+            params=PARAMS,
         )
         assert run.status.state == "completed"
         assert (
             run.status.results["databricks_runtime_task"]["logs"]
             == "{'param1': 'value1', 'param2': 'value2'}\n"
         )
+
+    @pytest.mark.parametrize(
+        "handler, databricks_code",
+        [
+            # ("print_kwargs", PRINT_KWARGS),
+            (
+                "",
+                PRINT_KWARGS.replace("print_kwargs", "handler"),
+            ),  # test default handler.
+        ],
+    )
+    def test_rerun(self, handler, databricks_code):
+        function_kwargs = {"handler": handler} if handler else {}
+        function_ref = FunctionReference(
+            kind="databricks",
+            code=databricks_code,
+            image="mlrun/mlrun",
+            name="databricks-test",
+        )
+
+        function = function_ref.to_function()
+
+        self._add_databricks_env(function=function, is_cluster_id_required=True)
```

Comment:
Is cluster ID really required? Seems to me that we need to do something with this parameter, which is set to `True` in all the tests. Why not drop it and use the cluster ID if provided and let a new cluster be created otherwise?

Reply:
I want to raise an error if it is missing from a test that requires it.

I understand the motivation behind your comment; I can set a default value of `is_cluster_id_required=True`.

Reply:
That's not the point though... The issue is that the test runs just fine with `is_cluster_id_required=False`, doesn't it? Which would mean that it's not actually required. Not related at all to whether the parameter has a default or not.

Reply:
I guess it doesn't have to be resolved in this PR though.